### DISCUSSION TITLE
Failure during initialization of keycloak-js adapter     at dsb-vue-keycloak.esm.js:162:23

### DISCUSSION THREAD

**Discussion by bieshaoxiong** [keycloak/keycloak#15183] [Q&A] [answered]
Failure during initialization of keycloak-js adapter
    at dsb-vue-keycloak.esm.js:162:23
![image](https://user-images.githubusercontent.com/64937534/198186131-c637dc74-62c1-4e46-baae-363e6b7f8366.png)
![image](https://user-images.githubusercontent.com/64937534/198186213-06934f72-1731-4758-8bcd-9ced66bfb73a.png)
请问这个需要怎么解决

**Comment by MemoryShadow** [comment, marked as answer]
You should go to the corresponding warehouse to find out.

look [#93 ](https://github.com/dsb-norge/vue-keycloak-js/issues/93)
